hayeon/programmers-2/쿼드압축-후-개수-세기.js

```diff
@@ -0,0 +1,39 @@
+function solution(arr) {
+  const answer = [0, 0];
+
+  function check(x, y, n) {
+    if (n === 1) return answer[arr[x][y]]++;
+
+    let flag = true;
+    for (let i = x; i < x + n; i++) {
+      for (let j = y; j < y + n; j++) {
+        if (arr[x][y] !== arr[i][j]) {
+          flag = false;
+          break;
+        }
+      }
+    }
+
+    if (flag) return answer[arr[x][y]]++;
+
+    n /= 2;
```

Comment:
변수로 관리빼서 관리하는게 좋을 것 같습니다.
```suggestion
const halfSize = n / 2;
```